iframe/index.html

```diff
@@ -0,0 +1,27 @@
+<!-- THIS IS AN EXAMPLE PAGE AND SHOULD NOT BE USED OUTSIDE OF DEVELOPMENT -->
```

Comment:
is this used to build the `iframe`? or is this just an example?
If it's an example maybe we can move it to an `examples/` directory or something?

Reply:
I tried to move it to `examples/` but the vite config kept failing and wouldn't resolve the `index/html`.